### PR TITLE
button bar pattern updates

### PR DESCRIPTION
**Pull request by Fercas123**

### CODE REVIEW

File: site/public/img/patterns/button-bar/button-bar-anatomy.png
Comment (joshwooding) [inline site/public/img/patterns/button-bar/button-bar-anatomy.png:1]: This image doesn't look right? @navkaur76

File: site/docs/patterns/button-bar.mdx [hunk 1]
```diff
@@ -62,72 +62,64 @@ A button bar typically contains:
...
 - This is commonly a related task or function that’s lower priority for the task, such as **Import** or **Export**.
 - You can also use it to signify a destructive action that may result in a permanent loss of data. This is commonly followed by user confirmation, e.g., a dialog. Examples include **Delete** or **Reset**.
 
-Remember to use a hierarchy when making your button choices, much like following a type ramp. In Example 1 below, Cancel is the alternate path and uses a primary button because there are only two options. However, in Example 2, Cancel uses the secondary button because three actions are available to the user, with this one being tertiary and the least desired option.
+Remember to use a hierarchy when making your button choices, much like following a type ramp. In Example 1 below, Cancel is the alternate path and uses a solid button because there are only two options. However, in Example 2, Cancel uses the transparent button because three actions are available to the user, with this one being tertiary and the least desired option.
 
-In the below example:
+### Example 1
```
Comment (joshwooding) [inline site/docs/patterns/button-bar.mdx:87]: I guess the example images haven't been updated yet? @Fercas123

File: site/docs/patterns/button-bar.mdx [hunk 2]
```diff
@@ -235,13 +223,13 @@ In a stacked button layout, we recommend buttons stack in order of importance fr
   - If users will access your application on a mobile device, use a stacked button layout to ensure an adequate touch-area for the buttons.
 </ExampleContainer>
 
-### Layout
+### Building a Stacked Button Bar
```
Comment (joshwooding) [inline site/docs/patterns/button-bar.mdx:226]:
```suggestion
### Building a stacked button bar
```

File: site/docs/patterns/button-bar.mdx [hunk 3]
```diff
@@ -62,72 +62,64 @@ A button bar typically contains:
...
-In the below example:
+### Example 1
 
-    - The **Submit** primary action uses the CTA button
-    - The **Cancel** alternate action uses the primary button
+    - The **Submit** solid action uses the solid button.
+    - The **Cancel** alternate action uses the bordered button.
 
   <Diagram
     src="/img/patterns/button-bar/button-options-example-1.png"
```
Comment (joshwooding) [inline site/docs/patterns/button-bar.mdx:93]: Just a note, these images don't exist

File: site/public/img/patterns/button-bar/button-bar-anatomy.png
Comment (joshwooding) [inline site/public/img/patterns/button-bar/button-bar-anatomy.png:1]: This image is still wrong?

File: site/public/img/patterns/button-bar/button-bar-building-stacked-button-bar-1.png
Comment (joshwooding) [inline site/public/img/patterns/button-bar/button-bar-building-stacked-button-bar-1.png:1]: Was this image exported at the right size?

File: site/public/img/patterns/button-bar/button-bar-building-stacked-button-bar-2.png
Comment (joshwooding) [inline site/public/img/patterns/button-bar/button-bar-building-stacked-button-bar-2.png:1]: Was this image exported at the right size?

File: site/docs/patterns/button-bar.mdx [hunk 4]
```diff
@@ -150,14 +140,12 @@ The container and the type of scenario determine the position of actions (CTA, p
...
     alt="Example showing a dialog with a form and button bar underneath. The core actions are on the right."
-    caption="Example showing a dialog with a form and button bar underneath. The core actions are on the right."
-    contentPosition="bottom"
     border
   />
 
 <ExampleContainer type="positive" customPillText="Best practices">
   - Group core onward actions for the form to ensure they remain accessible on browser zoom (e.g., **Previous**, **Next**, **Cancel** or **Submit**).
-  - Display secondary actions that aren’t core to the workflow at the opposite end of the button bar.
+  - You should display transparent actions that aren’t core to the workflow at the opposite end of the button bar.
```
Comment (joshwooding) [inline site/docs/patterns/button-bar.mdx:148]: This doesn't feel equivalent. "secondary actions" are an actual thing "transparent actions" aren't.  "core actions" are used to describe the other type of actions.

File: site/docs/patterns/button-bar.mdx [hunk 5]
```diff
@@ -62,72 +62,64 @@ A button bar typically contains:
...
 
-Remember to use a hierarchy when making your button choices, much like following a type ramp. In Example 1 below, Cancel is the alternate path and uses a primary button because there are only two options. However, in Example 2, Cancel uses the secondary button because three actions are available to the user, with this one being tertiary and the least desired option.
+Remember to use a hierarchy when making your button choices, much like following a type ramp. In Example 1 below, Cancel is the alternate path and uses a solid button because there are only two options. However, in Example 2, Cancel uses the transparent button because three actions are available to the user, with this one being tertiary and the least desired option.
 
-In the below example:
+### Example 1
 
-    - The **Submit** primary action uses the CTA button
-    - The **Cancel** alternate action uses the primary button
+    - The **Submit** solid action uses the solid button.
```
Comment (joshwooding) [inline site/docs/patterns/button-bar.mdx:89]: "solid" action? the opposite is alternate which makes this confusing to me

File: site/docs/patterns/button-bar.mdx (hunk 4 above)
Comment (joshwooding) [reply site/docs/patterns/button-bar.mdx:148]: Also secondary action is already used previously.

File: site/docs/patterns/button-bar.mdx [hunk 6]
```diff
@@ -62,72 +62,64 @@ A button bar typically contains:
     border
   />
 
-Buttons can display text or icons (or both) to describe the action. See [`Button`](/salt/components/button)'s documentation for further guidance.
-
 ## Button options
 
-There are three button variants to choose from. Choosing the correct button depends on the context and is ultimately the choice of the application team. However, we recommend following these guidelines for consistent design:
+There are three button appearances to choose from. Choosing the correct button depends on the context and is ultimately the choice of the application team. For guidance on button choices, please see the [`Button`](/salt/components/button) docs.  We recommend using the accented button set for a majority of use cases, alongside the following guidelines for consistent design:
```
Comment (joshwooding) [inline site/docs/patterns/button-bar.mdx:67]:
```suggestion
There are three button appearances to choose from. Choosing the correct button depends on the context and is ultimately the choice of the application team. For guidance on button choices, please see the [`Button`](/salt/components/button) docs. We recommend using the accented button set for a majority of use cases, alongside the following guidelines for consistent design:
```

**Review by joshwooding — CHANGES_REQUESTED** [review]

File: site/public/img/patterns/button-bar/button-bar-building-stacked-button-bar-1.png
Comment (origami-z) [inline site/public/img/patterns/button-bar/button-bar-building-stacked-button-bar-1.png:1]: needs 2x, image blurred

File: site/public/img/patterns/button-bar/button-bar-building-stacked-button-bar-2.png
Comment (origami-z) [inline site/public/img/patterns/button-bar/button-bar-building-stacked-button-bar-2.png:1]: 2x

**Review by origami-z — COMMENTED** [review]
Should button bar pattern code in storybook be updated?

**Review by mark-tate — DISMISSED** [review]

**Review by origami-z — APPROVED** [review]

File: site/docs/patterns/button-bar.mdx [hunk 7]
```diff
@@ -62,72 +66,75 @@ A button bar typically contains:
...
+**Transparent**
+- Use for the tertiary functions you're offering the user.
+- There can be multiple equally weighted tertiary functions within a button bar.
+- This is commonly a related task or function that’s lower priority than the secondary functions, such as **Import** or **Export**.
 
-Remember to use a hierarchy when making your button choices, much like following a type ramp. In Example 1 below, Cancel is the alternate path and uses a primary button because there are only two options. However, in Example 2, Cancel uses the secondary button because three actions are available to the user, with this one being tertiary and the least desired option.
+Remember to use a hierarchy when making your button choices, much like following a type ramp. In Example 1 below, Cancel is the alternate function and uses a solid button because there are only two options. However, in Example 2, Cancel uses the transparent button because three actions are available to the user, with this one being tertiary and the least desired option.
 
-In the below example:
+### Example 1
```
[the remark this answers is not in the document]
Comment (Fercas123) [reply site/docs/patterns/button-bar.mdx:90]: highlighted this on the spec to be looked at and followed up

**Pull request merged**